src/libcore/sync/atomic.rs

```diff
@@ -124,7 +124,7 @@ pub fn spin_loop_hint() {
 /// [`bool`]: ../../../std/primitive.bool.html
 #[cfg(target_has_atomic = "8")]
 #[stable(feature = "rust1", since = "1.0.0")]
-#[repr(transparent)]
+#[repr(align(1))]
```

Comment:
`align(1)` doesn't do anything here, but `repr(transparent)` does, so I'd opt to use that here.

Reply:
Yeah, this was only added for consistency. I don't know if `#[repr(transparent)]` is really appropriate for  `AtomicBool` though because the internal type is actually `u8` and if we wanted to support platforms that don't have byte level atomics it would have to be something bigger.